plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -1287,17 +1287,10 @@ def get_contract_instance(
             )
         return instance
 
-    def get_deploy_transaction(  # pylint: disable=arguments-differ
+    def get_deploy_transaction(  # pylint: disable=arguments-differ  # type: ignore
```

Comment:
why `type: ignore` added?, also probably now `disable=arguments-differ` can be removed